src/markdown_renderers.js

```diff
@@ -49,3 +53,98 @@ export const customHeadingRenderer = (text, level, raw) => {
             <h${level} id="${headingId}"><a href="#${headingId}"></a>${headingText}</h${level}>`;
     return headingToReturn;
 };
+
+export const parseRepoName = (repoUrl) => {
+    // Can't use parsedRepoUrl.full_name on it's own as Bitbucket adds irrelevant path suffix to the end of it
+    const parsedRepoUrl = gitUrlParse(repoUrl);
+    const repoNameParts = parsedRepoUrl.full_name.split('/');
+    const repoFullName = `${repoNameParts[0]}/${repoNameParts[1]}`;
+    return repoFullName;
```

Comment:
Maybe something like this?
```
const parsedRepoUrl = gitUrlParse(repoUrl);
const repoName = parsedRepoUrl.full_name.split('/').slice(0,2).join('/');
return repoName;
```
The parse`RepoName` func name and the repo`FullName` return value don't match so it makes sense to do it like this. Or just return the result of the last command.
I don't really know how many parts are there in `repoNameParts` but probably still makes sense to use the slice and join...